CityOfDetroit/bloom
# 2819/get assistance page

# Pull Request Template

## Issue Overview

This PR addresses #2819


## Description

Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:


## Reviewer Notes:

## QA Notes:
Entry point: http://localhost:3000/ - any page

1. Notice Get Assistance page in navigation bar instead of Resources
2. Visit page (/get-assistance) and compare layout to Figma mockup
3. Click through links and ensure they link correctly

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
